api/KeyboardEvent.json

```diff
@@ -1047,10 +1047,10 @@
                 "version_added": true
               },
               "safari": {
-                "version_added": null
+                "version_added": false
```

Comment:
Confirmed by "FIXME: The specification also has Fn, FnLock, Hyper, NumLock, Super, ScrollLock, Symbol, SymbolLock." comment.